api-docs/cloud-serverless/ref.yml

```diff
@@ -13864,18 +13864,27 @@ paths:
 
         Use this endpoint to send data in [line protocol](/influxdb/cloud-serverless/reference/syntax/line-protocol/) format to InfluxDB.
 
-        InfluxDB does the following when you send a write request:
+        InfluxDB Cloud Serverless does the following when you send a write request:
 
-        1. Validates the request
-        2. If successful, attempts to [ingest the data](/influxdb/cloud-dedicated/reference/internals/durability/#data-ingest); _error_ otherwise.
-        3.  If successful, responds with _success_ (HTTP `204` status code), acknowledging that the data is written and queryable; _error_ otherwise.
+        1.  Validates the request.
+        2.  If successful, attempts to [ingest the data](/influxdb/cloud-serverless/reference/internals/durability/#data-ingest); [error](/influxdb/cloud-serverless/write-data/troubleshoot/#review-http-status-codes) otherwise.
+        3.  Processes data in the request body and then responds with one of the following HTTP status codes:
+          - `204 Success`: All points in the batch have been written and are queryable.
+          - `400 Bad request`: Some points were rejected. The response body contains details about the [rejected points](/influxdb/cloud-serverless/write-data/troubleshoot/#troubleshoot-rejected-points). Points that were written are queryable.
```

Comment:
201 when partial-writes are on, and some lines were written.
Otherwise, still a 400.